src/plugins/home/public/application/components/guided_onboarding/use_case_card.test.tsx

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { UseCaseCard } from './use_case_card';
+
+jest.mock('../../kibana_services', () => ({
+  getServices: () => ({
+    application: {
+      navigateToApp: () => {},
+    },
+    trackUiMetric: () => {},
+  }),
+}));
+describe('use case card', () => {
```

Comment:
We need to add Storybook to this plugin.

Reply:
Honestly, **need** is a strong word. I wouldn't block any PR's that don't have storybook yet.

Reply:
If that's okay with you @clintandrewhall @cchaos, I'd like to add storybook in a separate PR to merge this one and unblock some other work for the project.

Reply:
@cchaos @yuliacech to clarify, that was a note for me, not for this PR 😄